src/components/select/UISelect.vue

```diff
@@ -249,43 +251,51 @@ export default defineComponent({
     emits: ['update:modelValue'],
 
     setup(props) {
+        const stringOptions = computed(() => props.options.every(option => typeof option === 'string'));
```

Comment:
This looks more complicated to me than just `typeof var ==='string'` which you already do in one place.
Consider removing?

Reply:
Not a fan of this either. It was only really needed in `select()`, so I've moved the check into there instead.